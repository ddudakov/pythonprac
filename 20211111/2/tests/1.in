q1fggyjsahu%61q8
(1,2), (4,5), (9,8)
