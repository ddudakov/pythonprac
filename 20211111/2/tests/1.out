Invalid input
3.00
